Makefile

```diff
@@ -120,11 +120,19 @@ clear-templates:
 .PHONY: clear-templates
 
 test-caching-service-manually:
-	oc process caching-service -p NAMESPACE=$(shell oc project -q) | oc create -f -
+	oc process caching-service -p NAMESPACE=$(shell oc project -q) -p APPLICATION_USER=test -p APPLICATION_USER_PASSWORD=test -p IMAGE=$(_IMAGE) | oc create -f -
+	oc expose svc/caching-service-app-http || true
```

Comment:
Exposing a service means creating a route. Is that what you wanted?

Reply:
Yes. Creating a route, especially for the HTTP endpoint makes it easier for testing locally using CURL etc